catalogue/webapp/components/IIIFViewer/ViewerSidebar.tsx

```diff
@@ -90,11 +96,7 @@ const AccordionItem = ({
   return (
     <Item data-test-id={testId}>
       <AccordionInner onClick={() => setIsActive(!isActive)}>
-        <button
-          className="plain-button no-margin no-padding"
-          aria-expanded={isActive ? 'true' : 'false'}
-          aria-controls={toHtmlId(title)}
-        >
+        <AccordionButton>
```

Comment:
Should the aria props still be passed in here?

Reply:
Oops, yes